examples/docker.yaml

```diff
@@ -63,3 +63,6 @@ probes:
         exit 1
       fi
     hint: See "/var/log/cloud-init-output.log". in the guest
+portForwards:
+  - guestSocket: "/run/user/{{.UID}}/docker.sock"
+    hostSocket: "{{.Home}}/docker.sock"
```

Comment:
> 5: # Hint: To allow `docker` CLI on the host to connect to the Docker daemon running inside the guest,

Could you update this part too?

Reply:
Do we want to update `podman.yaml` too?

Reply:
> Do we want to update `podman.yaml` too?

Fine with me; just let me know. I've updated `docker.yaml` because I used it for testing the PR. 😄

Reply:
> Do we want to update `podman.yaml` too?

I tried, but it doesn't work when I set `DOCKER_HOST=$HOME/podman.sock`, and neither does `PODMAN_HOST` or `CONTAINER_HOST`.

I have some suspision that `podman` is trying to be too smart, and knows that it can't be running natively on macOS, so only connects over ssh, or something.

Maybe @afbjorklund knows and can fix this in a followup PR?

Reply:
> `DOCKER_HOST=$HOME/podman.sock`

Did you mean `DOCKER_HOST=unix://$HOME/podman.sock`

Reply:
> > `DOCKER_HOST=$HOME/podman.sock`
> 
> Did you mean `DOCKER_HOST=unix://$HOME/podman.sock`

Yes:

```console
$ CONTAINER_HOST=ssh://$(id -un)@localhost:60906/run/user/$(id -u)/podman/podman.sock CONTAINER_SSHKEY=$HOME/.lima/_config/user podman ps
CONTAINER ID  IMAGE       COMMAND     CREATED     STATUS      PORTS       NAMES

$ DOCKER_HOST=unix://$HOME/podman.sock podman ps
Cannot connect to Podman. Please verify your connection to the Linux system using `podman system connection list`, or try `podman machine init` and `podman machine start` to manage a new Linux VM
Error: unable to connect to Podman socket: Get "http://d/v3.4.1/libpod/_ping": dial unix ///var/folders/wm/pq6w7kf54215fxz392ff3rxr0000gn/T/podman-run--1/podman/podman.sock: connect: no such file or directory

$ ls -l ~/*.sock
srw-------  1 jan  staff  0  1 Nov 17:47 /Users/jan/podman.sock
```

Reply:
Hold on, there is a typo in the forward definition 😞

Reply:
Yes, typo. Now works with `DOCKER_HOST=... docker ps` as well as `CONTAINER_HOST=... podman ps`.

I've updated the PR